src/azure-cli/setup.py

```diff
@@ -86,7 +86,7 @@
     'azure-mgmt-eventgrid==10.2.0b2',
     'azure-mgmt-eventhub~=10.1.0',
     'azure-mgmt-extendedlocation==1.0.0b2',
-    'azure-mgmt-hdinsight~=9.0.0',
```

Comment:
the next beta version should be 9.0.1b1 as you already released the 9.0.0 stable version